draft-ietf-httpbis-message-signatures.md

```diff
@@ -480,6 +480,7 @@ The signature parameters component value is the serialization of the signature p
 * `nonce`: A random unique value generated for this signature as a String value.
 * `alg`: The HTTP message signature algorithm from the HTTP Message Signature Algorithm Registry, as a String value.
 * `keyid`: The identifier for the key material as a String value.
+* `context`: A context for the signature as a String value. This value is used by applications to help identify relevant signatures.
```

Comment:
```suggestion
* `context`: A context for the signature as a String value. This value is used by applications to help identify the signature's validity in specific applications or protocols. Context is often needed for avoiding cross-protocol attacks.
```

IMO the value does not identify the signature (which the "sig1" name is doing), rather it clarifies within which context the signature is valid.